src/builtin-addons/core/intl-completion-provider.ts

```diff
@@ -54,7 +54,7 @@ export default class IntlCompletionProvider {
                 end: endPosition,
               },
             },
-            detail: detail,
+            documentation: detail,
```

Comment:
are we sure on field change?

Reply:
This is how it looks with `detail` field:

![completionDetails](https://user-images.githubusercontent.com/18197844/141479205-e0555b37-95cd-4b77-a774-d27d9b2ef32e.gif)
and with `documentation` field:

![completionDocumentation](https://user-images.githubusercontent.com/18197844/141479242-2b00ea42-11f0-4e10-8438-83051228cd5b.gif)

While the first one provide quicker preview of translation it fails if we show something longer
![completionLonger](https://user-images.githubusercontent.com/18197844/141479799-264b8e30-fe70-423b-bf2a-9055f8a35161.gif)

I can leave it as before, but as hovers support multiline text, then I wanted to have it displayed the same way in completions

Reply:
agree, let's keep your changes!